osu.Game/Screens/Select/MatchSongSelect.cs

```diff
@@ -17,6 +22,15 @@ public class MatchSongSelect : SongSelect, IMultiplayerSubScreen
         public string ShortTitle => "song selection";
         public override string Title => ShortTitle.Humanize();
 
+        [Resolved(typeof(Room))]
+        protected Bindable<PlaylistItem> CurrentItem { get; private set; }
+
+        [Resolved]
+        protected Bindable<IEnumerable<Mod>> CurrentMods { get; private set; }
```

Comment:
`private` + rename this to `SelectedMods` (kinda sounds like "current" room mods)